app/src/main/java/com/example/bigowlapp/viewModel/SupervisedGroupListViewModel.java

```diff
@@ -19,11 +21,19 @@ public SupervisedGroupListViewModel() {
         groupRepository = new GroupRepository();
     }
 
-    public MutableLiveData<List<Group>> setListOfDocumentByArrayContains() {
-        groupLiveData = groupRepository.getListOfDocumentByArrayContains("supervisedUserId",
-                authRepository.getCurrentUser().getUid(), Group.class);
+    public MutableLiveData<List<Group>> getSupervisedGroupList() {
```

Comment:
Super small thing, but should return this as LiveData not MutableLiveData

Reply:
Makes sense. Consider it done.